rustls/src/tls12/mod.rs

```diff
@@ -29,7 +29,9 @@ pub struct Tls12CipherSuite {
     /// How to sign messages for authentication.
     pub sign: &'static [SignatureScheme],
 
-    pub(crate) aead_alg: &'static dyn Tls12AeadAlgorithm,
+    /// Producing a suitable `MessageDecrypter` or `MessageEncrypter`
+    /// from the raw keys.
```

Comment:
nit: maybe match the pattern used by the other docstrings and say "How to produce a ... "